frontend/package.json

```diff
@@ -26,7 +26,7 @@
     "lint": "eslint --ext .ts,.js --ignore-path .gitignore .",
     "lint:fix": "eslint --fix --ext .ts,.js --ignore-path .gitignore ."
   },
-  "proxy": "http://localhost:8080",
+  "proxy": "http://app-server:8080",
```

Comment:
This forces docker compose to be used.

Reply:
For now I guess we allow it to run only using docker-compose because multi-environment still has some issues